src/main/kotlin/org/ghrobotics/frc2019/subsystems/Drivetrain.kt

```diff
@@ -41,10 +47,10 @@ object Drivetrain : FalconWestCoastDrivetrain() {
     // Overriden variables
 
     // Motors
-    override val leftMotor: FalconMotor<Meter> = TODO()
-    override val rightMotor: FalconMotor<Meter> = TODO()
-    private val leftSlave1: FalconMotor<Meter> = TODO()
-    private val rightSlave1: FalconMotor<Meter> = TODO()
+    override val leftMotor: FalconMAX<Meter> = FalconMAX(kLeftMasterId, CANSparkMaxLowLevel.MotorType.kBrushless, NativeUnitLengthModel(306.18.nativeUnits, 3.inch))
```

Comment:
Since you're using the same native unit model, just define a constant at the top of the file with the native unit model. Then, use that constant for all four motors. It would also be preferable to create two other constants, one for wheel radius and one for sensor units per rotation so that we don't have arbitrary magic numbers floating around.